crates/move-prover-boogie-backend/src/boogie_backend/bytecode_translator.rs

```diff
@@ -703,6 +721,54 @@ impl<'env> BoogieTranslator<'env> {
         }
     }
 
+    fn translate_function_no_abort(&self, fun_env: &FunctionEnv) {
+        let style = FunctionTranslationStyle::SpecNoAbortCheck;
+        let variant = FunctionVariant::Baseline;
+        use Bytecode::*;
+
+        let requires_function =
+            Operation::apply_fun_qid(&fun_env.module_env.env.requires_qid(), vec![]);
+        let ensures_function =
+            Operation::apply_fun_qid(&fun_env.module_env.env.ensures_qid(), vec![]);
+        let asserts_function =
+            Operation::apply_fun_qid(&fun_env.module_env.env.asserts_qid(), vec![]);
+        let ensures_asserts_to_requires_subst = BTreeMap::from_iter(vec![
+            (ensures_function.clone(), requires_function.clone()),
+            (asserts_function.clone(), requires_function.clone()),
+        ]);
```

Comment:
`requires`/`ensures`/`asserts` shouldn't appear in regular functions, so these shouldn't be required